src/compiler/emitter.ts

```diff
@@ -2050,6 +2050,67 @@ module ts {
             function getTemplateLiteralAsStringLiteral(node: LiteralExpression): string {
                 return '"' + escapeString(node.text) + '"';
             }
+            
+            function emitDownlevelRawTemplateLiteral(node: LiteralExpression, isLast: boolean) {
+                var text = getSourceTextOfNodeFromSourceFile(currentSourceFile, node);
+                
+                // text contains the original source, it will also contain quotes ("`"), dolar signs and braces ("${" and "}"),
+                // thus we need to remove those characters.
+                // First template piece starts with "`", others with "}"
+                // Last template piece ends with "`", others with "${"
+                text = text.substring(1, text.length - (isLast ? 1 : 2));
+                
+                write('"' + escapeString(text) + '"');

```

Comment:
Newline normalization still takes place here as per the spec:

> The TRV of *LineTerminatorSequence* :: \<CR> is the code unit value 0x000A.
> The TRV of *LineTerminatorSequence* :: \<CR>\<LF> is the sequence consisting of the code unit value 0x000A.

So you'll need to replace `\r\n`s and `\r`s with `\n`s.

I'd do something like

```TypeScript
text = text.replace(/\r\n?/g, "\n");
text = escapeString(text);

write('"' + text + '"');
```

I think we should have tests for a lot of this.